### PR TITLE
Specify radix on integer parsing

### PR DESCRIPTION
**Pull request by bitfriend**
Will fix this exception on flutter side:
`FormatException: Invalid radix-10 number`

### CODE REVIEW

**Comment by bitfriend** [comment]
This was not error before.
But it seems to be error now with upgraded flutter.

**Comment by bitfriend** [comment]
When opening any room, you will get that exception.

**Review by gnunicorn — APPROVED** [review]

File: app/lib/common/utils/utils.dart [hunk 1]
```diff
@@ -129,7 +129,8 @@ int hexOfRGBA(int r, int g, int b, {double opacity = 1}) {
   b = (b > 255) ? 255 : b;
   int a = opacity.toInt();
   return int.parse(
-    '0x${a.toRadixString(16)}${r.toRadixString(16)}${g.toRadixString(16)}${b.toRadixString(16)}',
+    '${a.toRadixString(16)}${r.toRadixString(16)}${g.toRadixString(16)}${b.toRadixString(16)}',
+    radix: 16,
```
[the remark this answers is not in the document]
Comment (bitfriend) [reply app/lib/common/utils/utils.dart:133]: I couldn't get that exception with this solution.

**Review by gtalha07 — APPROVED** [review]
Looks good!

**Pull request merged**